### PR TITLE
Ikoevska/ns 120/changelog

### PR DESCRIPTION
**Pull request by ikoevska**
-Fixed instances of non-existent list-devices command
-Fixed documentation for livesync when no platform is specifies
-Added changelog entries for 1.2.0
-Updated plugins.md with support for native libraries and "folders as modules" functionality.

### CODE REVIEW

**Comment by teobugslayer** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+==
+
+### New
+
+* [Implemented #621](https://github.com/NativeScript/nativescript-cli/issues/621): Added the `$ tns error-reporting` command. By default, anonymous error reporting is enabled.
+* [Implemented #620](https://github.com/NativeScript/nativescript-cli/issues/620): Renamed the `$ tns feature-usage-tracking` command to `$ tns usage-reporting`. By default, anonymous usage reporting is enabled.
+* [Implemented #523](https://github.com/NativeScript/nativescript-cli/issues/523): Added the `$ tns livesync <Platform>` command. You can use it to quickly synchronize changes to connected devices without re-building and re-deploying your apps.
+* [Implemented #510](https://github.com/NativeScript/nativescript-cli/issues/510): Improvements and updates to the `$ tns plugin` sets of commands.
+	* You can create and work with custom NativeScript plugins which contain Android native libraries.
+	* You can create and work with custom NativeScript plugins which containg iOS dynamic native libraries.
```
Comment (ligaz) [inline CHANGELOG.md]: contain

**Comment by dtopuzov** [comment]
:+1:

**Comment by Fatme** [comment]
:+1:

**Pull request merged**